packages/cli/src/config/schema.ts

```diff
@@ -173,9 +173,15 @@ export const schema = {
 			enableWAL: {
 				doc: 'Enable SQLite WAL mode',
 				format: Boolean,
-				default: false,
+				default: true,
```

Comment:
Should this be marked as a breaking change, considering that WAL has some downsides, e.g. the file system has to support it, it's a bit slower and backup solutions need to be aware of additional files: 
https://www.sqlite.org/wal.html

Reply:
updated this to be backward-compatible. now WAL is always enabled when using pooling, but by default, the env variable is respected.